loony_dev/agents/coding.py

```
@@ -88,13 +88,30 @@ def execute_issue(self, task: IssueTask) -> TaskResult:
         default_branch = GitRepo.detect_default_branch(self.work_dir)
         git = GitRepo(self.work_dir, default_branch=default_branch)
         session_id = self._session_id_for(task)
+        branch = task.branch_name
+
+        # Prepare the worktree: checkout branch if it exists, create it if not.
+        try:
+            git.checkout_or_create_branch(branch)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)